ui/components/app/whats-new-popup/whats-new-popup.js

```diff
@@ -33,38 +25,11 @@ import { updateViewedNotifications } from '../../../store/actions';
 import { ButtonPrimary, Text } from '../../component-library';
 import Popover from '../../ui/popover';
 
-function getActionFunctionById(id, history) {
+function getActionFunctionById(id) {
   const actionFunctions = {
-    8: () => {
-      updateViewedNotifications({ 8: true });
-      history.push(ADVANCED_ROUTE);
-    },
-    20: () => {
-      updateViewedNotifications({ 20: true });
-      global.platform.openTab({
-        url: ZENDESK_URLS.LEDGER_FIREFOX_U2F_GUIDE,
-      });
-    },
-    24: () => {
-      updateViewedNotifications({ 24: true });
-    },
     [NOTIFICATION_DROP_LEDGER_FIREFOX]: () => {
       updateViewedNotifications({ [NOTIFICATION_DROP_LEDGER_FIREFOX]: true });
     },
-    [NOTIFICATION_U2F_LEDGER_LIVE]: () => {
-      updateViewedNotifications({ [NOTIFICATION_U2F_LEDGER_LIVE]: true });
-    },
-    ///: BEGIN:ONLY_INCLUDE_IF(blockaid)
-    [NOTIFICATION_BLOCKAID_DEFAULT]: () => {
```

Comment:
Still present in tests

![Screenshot 2024-06-11 at 8 40 14 AM](https://github.com/MetaMask/metamask-extension/assets/8112138/f2ddc967-e30b-44bf-abe4-054047e5b00e)